src/scss/components/main-nav/_main-nav.scss

```diff
@@ -0,0 +1,64 @@
+@charset "UTF-8";
+
+.su-multi-menu {
+  .su-multi-menu__nav-toggle--right {
+    &:hover::before,
+    &::before {
+      color: transparent;
+      background-color: transparent;
+    }
+
+    &[aria-expanded="true"] {
+      color: #e50808;
```

Comment:
Colors should all be in variable names in the scss files. It should auto complete in php storm for you.

If there is ever a color not in there or our saa colors file we can add it to the saa colors file.